bin/i18n/deps.edn

```diff
@@ -4,6 +4,7 @@
  {common/common                     {:local/root "../common"}
   cheshire/cheshire                 {:mvn/version "5.8.1"}
   clj-http/clj-http                 {:mvn/version "3.9.1"}
+  io.github.borkdude/grasp          {:mvn/version "0.0.3"}
```

Comment:
I understand that grasp is a good library and borkdude is a trustable, highly competent dude. However, seeing '0.0.3' makes me terrified anyways - and presumably third parties would be more terrified